setup.cfg

```diff
@@ -69,6 +69,8 @@ microsoft.azure =
     apache-airflow-providers-microsoft-azure
 snowflake =
     apache-airflow-providers-snowflake
+dbt =
```

Comment:
this need to be in alphabetical order. Move it after databricks please

Reply:
Corrected